hbase-assembly/src/main/assembly/client.xml

```diff
@@ -67,6 +67,13 @@
               <exclude>io.opentelemetry.javaagent:*</exclude>
               <exclude>org.hamcrest:hamcrest-core</exclude>
               <exclude>org.mockito:mockito-core</exclude>
+              <!-- Exclude transitive dependencies of tomcat-jasper, not needed at runtime -->
```

Comment:
Kept only minimal things with which was able to open web UI do avoid polluting assembly.